### ISSUE TITLE
Open Data Licence

### ISSUE THREAD

**Issue by JLWade** [bcgov/parks-data-register#123]
https://www2.gov.bc.ca/assets/gov/data/open-data/open_data_assessment_and_checklist-update.docx

**Comment by JLWade** [comment]
https://catalogue.data.gov.bc.ca/

**Comment by JLWade** [comment]
no update :(
I need to reach out to a couple folks to get this moving. Wonder if I should remove form sprint and add it to a future one where I feel there is capacity to do it instead of continuing to carry it over?

**Comment by Dianadec** [comment]
@JLWade yeah that sounds good. We can work together to define this ticket a bit more to make it feel more achievable :)

**Comment by JLWade** [comment]
@kgorana to set up time with Jess to go over checklist and next steps

**Comment by kgorana** [comment]
moving to blocked until we hear back from the three teams that need to be contacted:
- Privacy Officer
- Information Security Officer (MISO)
- Intellectual Property Program

**Comment by Dianadec** [comment]
@JLWade could you provide an update on this one?
